demo/src/components/Cart/Cart.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {Container, Col, Row, Form} from 'react-bootstrap';
+import './Cart.css';
+import {setQuantity} from '../../store/StoreHelpers';
+import Image from 'react-bootstrap/Image';
+import PropTypes from 'prop-types';
+
+/**
+ * Creates a component describing a shopping Cart.
+ * @param {!Object} state The site state
+ * @param {function} setQuantity function to modify quantity of items
+ *      in the global state
+ * @return {!JSX} The component.
+ */
+const CartBase = function Cart({items, setQuantity}) {
+  const itemsRender = [];
```

Comment:
const /** !Array<!JSX> */ itemsRender = [];

https://google.github.io/styleguide/jsguide.html#jsdoc-template-parameter-types
Always specify template parameters. This way compiler can do a better job and it makes it easier for readers to understand what code does.

Reply:
Fixed